ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -184,6 +184,7 @@ Item {
             height: userImage.height
             width: 318
             Layout.alignment: Qt.AlignHCenter
+            Layout.topMargin: 33
```

Comment:
@faust4exe this also... (and if it's possible set it to 32, 33 looks somehow weird)